test/webpack.config.js

```diff
@@ -5,6 +5,154 @@ const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPl
 
 const ALIASES = require('../aliases')('src');
 
+const LIB_DIR = resolve(__dirname, '..');
+const SRC_DIR = resolve(LIB_DIR, './modules');
+
+function makeLocalDevConfig(EXAMPLE_DIR = LIB_DIR) {
```

Comment:
What is this and why do you need it? Render test and browser test are both run against source. Reuse the webpack config from them.

Reply:
@Pessimistress updated